File: src/main/java/com/josdem/vetlog/Configuration/SwaggerConfig.java
package com.josdem.vetlog.Configuration;

import io.swagger.v3.oas.models.info.Info;
import io.swagger.v3.oas.models.OpenAPI;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class SwaggerConfig {

    @Bean
    public OpenAPI vetlogOpenAPI() {
        return new OpenAPI()
                .info(new Info()
                        .title("Vetlog API")
                        .description("API documentation for Vetlog backend")
                        .version("1.0"));

    }
}

File: src/main/java/com/josdem/vetlog/configuration/SwaggerConfig.kt
package com.josdem.vetlog.configuration

import io.swagger.v3.oas.models.OpenAPI
import io.swagger.v3.oas.models.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class SwaggerConfig {

    @Bean
    fun vetlogOpenAPI(): OpenAPI {
        return OpenAPI()
            .info(
                Info()
                    .title("Vetlog API")
                    .description("API documentation for Vetlog backend")
                    .version("1.0")
            )
    }
}
